### PR TITLE
Fs3 deals

### PR DESCRIPTION
**Pull request by GuohaoMa**
Add online deal api, update browser.

### CODE REVIEW

File: cmd/web-handlers.go [hunk 1]
```diff
@@ -2525,37 +2533,57 @@ func (d *DealVo) setDefault() {
...
+		return "", err
+	}
+	out_bytes, _ := ioioutil.ReadAll(stdout)
+	if err := stdout.Close(); err != nil {
+		fmt.Println("Execute failed when close stdout:" + err.Error())
+		return "", err
+	}
+	if err := cmd.Wait(); err != nil {
+		fmt.Println("Execute failed when Wait:" + err.Error())
+		//return "", err
```
Comment (zzhao-nbai) [inline cmd/web-handlers.go:2570]: 把注释去掉

File: cmd/web-handlers.go [hunk 2]
```diff
@@ -2525,37 +2533,57 @@ func (d *DealVo) setDefault() {
...
-	err := decoder.Decode(&dealVo)
+	var onlineDealRequest OnlineDealRequest
+	err := decoder.Decode(&onlineDealRequest)
 	if err != nil && err != io.EOF {
 		w.Write([]byte(fmt.Sprintf("bad request: %s", err.Error())))
 		return
 	}
-	dealVo.setDefault()
-
+	fmt.Println("hello1")
```
Comment (zzhao-nbai) [inline cmd/web-handlers.go]: remove:print hello

Comment (GuohaoMa) [reply cmd/web-handlers.go]: already removed. Maybe the code line is not updated by github.

File: cmd/web-handlers.go (hunk 1 above)
Comment (GuohaoMa) [reply cmd/web-handlers.go]: Done.

File: cmd/web-handlers.go [hunk 3]
```diff
@@ -2574,171 +2602,68 @@ func (web *webAPIHandlers) SendDeal(w http.ResponseWriter, r *http.Request) {
...
-	bodyBytes, _ := io.ReadAll(res.Body)
+    // send online deal to lotus
+    //filWallet := os.Getenv("fil_wallet")
 
-	err = json.Unmarshal(bodyBytes, &dealResponse)
+	commandLineWallet := "echo $fil_wallet"
+	filWallet,err:= ExecCommand(commandLineWallet)
 	if err != nil {
-		return nil, err
+		fmt.Println(err)
```
Comment (zzhao-nbai) [inline cmd/web-handlers.go:2622]: error need to be recorded in log

File: cmd/web-handlers.go [hunk 4]
```diff
@@ -2574,171 +2602,68 @@ func (web *webAPIHandlers) SendDeal(w http.ResponseWriter, r *http.Request) {
...
-
-	return dealResponse.Data, nil
+	fmt.Println(filWallet)
+
+	verifiedDeal := "--verified-deal="+ onlineDealRequest.VerifiedDeal
+    fastRetrieval := "--fast-retrieval="+ onlineDealRequest.FastRetrieval
+	commandLine := "lotus "+"client "+ "import " + sourceFilePath
+	dataCID,err := ExecCommand(commandLine)
+    if err != nil {
+		fmt.Println(err)
```
Comment (zzhao-nbai) [inline cmd/web-handlers.go:2629]: should be in logs file

**Review by zzhao-nbai — APPROVED** [review]

**Pull request merged**